### PR TITLE
Sync to latest commits for manta dependencies

### LINKED ISSUE #64: Test Coverage for all repos.
- [ ] manta-node
     * [ ] service.rs
     * [ ] rpc.rs
     * [x] command.rs
- [ ] manta-app
- [ ] manta-api
     * [ ] circuit.rs
     * [ ] payload.rs
     * [ ] gadget.rs
- [ ] manta-subxt
- [x] manta-crypto
     * [x] commitment.rs
     * [x] checksum.rs
     * [x] merkle_tree.rs
- [x] manta-error
     * [x] lib.rs
- [x] manta-asset
     * [x] processing.rs
     * [x] serdes.rs
     * [x] sanity.rs
- [ ] manta-data
     * [ ] default.rs
     * [ ] metadata.rs
     * [ ] sanity.rs
     * [ ] serdes.rs
     * [ ] zkp.rs
- [ ] manta-ledger
     * [ ] lib.rs

### PR DESCRIPTION
**Pull request by ghzlatarev**
closes Manta-Network/Manta#64
 
* Update manta dependencies to latest commits
* Update CI

### CODE REVIEW

**Review by zhenfeizhang — APPROVED** [review]

**Comment by ghzlatarev** [comment]
Yo @stechu can we merge this ?

EDIT: scratch that for now.

**Comment by ghzlatarev** [comment]
> Yo @stechu can we merge this ?
> 
> EDIT: scratch that for now.

We need to wait for https://github.com/Manta-Network/manta-api/pull/15 and update ``manta-api`` to get rid of std features in our runtime. That's why the tests are failing.

File: .github/workflows/check_self.yml [hunk 1]
```diff
@@ -48,14 +48,14 @@ jobs:
...
       - name: Check Clippy
         run: |
-          cargo clippy --release --all
+          cargo clippy --all
       - name: Check Build
         run: |
-          cargo check --release
-          cargo +nightly check --release --features runtime-benchmarks
+          cargo check
+          cargo check --all-features
```
Comment (stechu) [inline .github/workflows/check_self.yml:58]: are these redundant checks?
`cargo check` and `cargo check --all-features` ?

**Comment by stechu** [comment]
Please also update to new dependencies.

File: .github/workflows/check_self.yml (hunk 1 above)
Comment (ghzlatarev) [reply .github/workflows/check_self.yml:58]: No, the latter add the runtime-benchmark features. They compile more stuff.

**Comment by ghzlatarev** [comment]
> Please also update to new dependencies.

Updated.

File: .github/workflows/check_self.yml (hunk 1 above)
Comment (stechu) [reply .github/workflows/check_self.yml:58]: @ghzlatarev  sorry for the late reply. I mean can we just use `cargo check --all-features`.

Comment (ghzlatarev) [reply .github/workflows/check_self.yml:58]: I've had situations where it compiled in ``--all-features`` but not without it.
Can't remember what it was exactly, but that's why I kept both.

**Pull request closed without merging**